### PR TITLE
DemandCard dateFormatter extracted to Extension, "Telefon Numarası" TextFormField Formatter

### PR DESCRIPTION
**Pull request by bmercan**
lib/pages/my_demand_page/my_demand_page.dart line 48 was expecting coma, nothing change other than that file

### CODE REVIEW

**Comment by bmercan** [comment]
**About Conflict**
lib/pages/my_demand_page/my_demand_page.dart line 48 was expecting coma, nothing change other than that file

File: lib/shared/extensions/date_count_down_extension.dart [hunk 1]
```diff
@@ -0,0 +1,14 @@
+extension DateCountDown on DateTime {
+  String get countDown {
```
Comment (easeccy) [inline lib/shared/extensions/date_count_down_extension.dart:2]: Rename `asElapsedTimeString`.

File: lib/pages/auth_page/auth_page.dart [hunk 2]
```diff
@@ -102,6 +103,14 @@ class _AuthPageState extends State<AuthPage> {
               ),
               const SizedBox(height: 28),
               TextFormField(
+                inputFormatters: [
+                  FilteringTextInputFormatter.allow(
+                    RegExp(
+                      '[+0-9]',
```
Comment (easeccy) [inline lib/pages/auth_page/auth_page.dart:109]: Diğer sayfalardaki inputlara da koy.

**Review by easeccy — CHANGES_REQUESTED** [review]
Apply fixes.

**Review by easeccy — APPROVED** [review]
LGTM

**Pull request merged**